membership-attribute-service/app/models/Attributes.scala

```diff
@@ -26,7 +26,8 @@ case class Attributes(
   MembershipJoinDate: Option[LocalDate] = None,
   DigitalSubscriptionExpiryDate: Option[LocalDate] = None,
   MembershipNumber: Option[String] = None,
-  AdFree: Option[Boolean] = None) {
+  AdFree: Option[Boolean] = None,
+  ActionAvailableFor: Option[String] = None) {
```

Comment:
is this really Notification rather than action? what does the "for" bit mean?

Reply:
https://github.com/guardian/frontend/pull/19176/files
Looks like frontend is actually expecting that /me will have 
`alertAvailableFor=membership` in the response if there needs to be a banner that directs the customer to the membership manage page.

Reply:
Oh, so I'm planning on changing this from ActionAvailableFor -> AlertAvailableFor unless calling it something other than AlertAvailableFor is worth the delay of me also making a change in frontend.